docs/make.jl

```diff
@@ -111,6 +111,10 @@ const PAGES = Any[
         "Installation" => "install.md",
         "Basics" => "basics.md",
         "Tutorial" => "tutorial.md",
+        "Series Types" => [
+            "Contour Plots" => "SeriesTypes/contour.md",
+            "Histograms" => "SeriesTypes/histogram.md",
```

Comment:
```suggestion
            "Contour Plots" => "series_types/contour.md",
            "Histograms" => "series_types/histogram.md",
```

Camel case is reserved for packages or module names (dirs would have to be renamed).